assets/js/search.js

```diff
@@ -45,7 +49,11 @@ window.addEventListener("DOMContentLoaded", () => {
   });
 
   window.addEventListener("keydown", (evt) => {
-    if (evt.key === "k" && evt.ctrlKey) {
+    if (
+      evt.key === "k" &&
+      ((isMac && evt.metaKey) || (!isMac && evt.ctrlKey))
```

Comment:
I know this feels like a silly tweak, but if you'll humor me: could we swap these two lines around to mimic the order in which the keystroke is presented to the user?

Reply:
Haha, makes sense to me! Addressed in c2de9fd666257229963a68f13ba773b834ca1da4, apologies for the force push.